README.md

```diff
@@ -519,6 +519,59 @@ The constant `acceleration` of the contact point. The default values are `0, 0,
 
 #### Loopconstraint
 
+#### Actuators
+The Actuators specifies the different parameters used to express the torque generated by a particular movement at a joint.
+In function of the type of actuator the parameters are different, in this example we choose the type Gauss3P and the parameters associate.
+These different parameters were defined based on the relationship between torque and angular velocity (`Tmax`, `T0`, `wmax`, `wc`), 
+the differential activation and angular velocity (`amin`, `wr`, `w1`), 
+and torque and angle (`qopt`, `r`) describe by M. Jackson (The mechanics of the table contact phase of gymnastics vaulting, 2019).
+```c
+actuator    default_segment
+            type    Gauss3p
+            dof     RotX
+            direction   positive
+            Tmax    220.3831
+            T0      157.4165
+            wmax    475.0000
+            wc      190.0000
+            amin    0.9900
+            wr      40.0000
+            w1     -90.0000
+            r       56.4021
+            qopt    25.6939
+        endactuator
+
+``` 
+#### type
+Different types of actuator can be defined with the `type` tag : `Constant`, `Linear`, `Gauss3P`, `Gauss6P`, `Sigmoidgauss3P`.
+#### dof
+The `dof` tag defines the degree of fredoom of the segment. It can be a rotation (`Rot`) or a translation (`Trans`) on one 
+of the 3 axis (`x`, `y`, `z`).
+#### direction
+The `direction` of the torque can be positive or negative.
```

Comment:
All of these are not required. Please either create a subsubsub section below "Constant", "Linear".... with the relevent tags, or specify in each tag for which Type they are relevent

Reply:
I have created subsubsub section for each parameters and indicated the types of actuators (i.e. Contant, Linear ...) that use this parameter